lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -92,6 +93,27 @@
 )
 
 
+def includes_image_modification(config):
```

Comment:
maybe this function can be moved to `image.modifier` (?)

Reply:
it's more a config-related function than an image modifier one, no?
would it  be better in `io.config`?

Reply:
Sure, agreed

Reply:
done